javascript/react/intro-react/what-is-react.md

```diff
@@ -29,7 +29,7 @@ aspects:
 
 The big and frequent problem that **React** solves is keeping the `DOM`[3] in sync with the data model.
 
-It does that by generating a `DOM`-like custom object whenever a change is made in the data model. This complex data type is then compared with the real `DOM`, updating the latter only when necessary, thus minimising execution time.
+It does that by generating a `DOM`-like custom object whenever a change is made in the data model. This complex data type is then compared with the real `DOM`, updating the latter only when necessary, thus minimizing execution time.
```

Comment:
Add a footnote here about Virtual DOM.

See: https://reactjs.org/docs/faq-internals.html

Reply:
Added the following footnote:

[4:Virtual DOM]
This is also referred to as the Virtual DOM or VDOM. It represents a concept in which a "virtual" representation of the UI is kept in memory and synced with the "real" DOM by a libray. In our case, this library is `ReactDOM`.